benchmark_runner/common/oc/oc.py

```diff
@@ -648,6 +649,27 @@ def wait_for_initialized(self, label: str, workload: str = '', status: str = 'In
             else:
                 raise PodNotInitializedTimeout(workload=workload)
 
+    @typechecked
+    @logger_time_stamp
+    def wait_for_patch(self, pod_name: str, label: str, label_uuid: bool, namespace: str, timeout: int = SHORT_TIMEOUT):
+        """
+        This method waits for patch, needs to wait that pod is created and then wait for ready
+        @param pod_name:
+        @param label:
+        @param label_uuid:
+        @param namespace:
+        @param timeout:
+        @return:
+        """
+        current_wait_time = 0
+        while timeout <= 0 or current_wait_time <= timeout:
+            if self._get_pod_name(pod_name=pod_name, namespace=namespace) and self.wait_for_ready(label=label, label_uuid=label_uuid, namespace=namespace):
+                return True
+            # sleep for x seconds
+            time.sleep(OC.SLEEP_TIME)
+            current_wait_time += OC.SLEEP_TIME
```

Comment:
Rather than rely on OC.SLEEP_TIME here, use time.time() to get the actual time.

Reply:
We use sleep of 3 sec between each operation waiting that the patch pod will be started

Reply:
This isn't that big of a deal (if your timeout runs long, it's not really a problem), but don't assume that each cycle will take exactly 3 seconds.

Reply:
Its for short period tasks, like create pod.